packages/devtools_app/lib/src/framework/initializer.dart

```diff
@@ -78,7 +82,11 @@ class _InitializerState extends State<Initializer>
           !connectionState.userInitiatedConnectionState) {
         // Try to reconnect (otherwise, will fall back to showing the
         // disconnected overlay).
-        unawaited(_attemptUrlConnection());
+        try {
+          unawaited(_attemptUrlConnection());
```

Comment:
I don't think this try-catch does anything if you don't await `_attemptUrlConnection` since it'll throw an asynchronous error if it throws. You'll most likely want to use `Future.catchError` here instead.